docs/pt/Accessing-Alluxio-from-Zeppelin.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: global
+title: Acessando o Alluxio a partir do Zeppelin
+nickname: Apache Zeppelin
+group: Frameworks
+priority: 4
+---
+
+[Apache Zeppelin](http://http://zeppelin-project.org/) é uma plataforma `web` que permite a análise 
+de dados interativa. A versão anterior de Alluxio (Tachyon 0.8.2) foi 
+[integrada](https://github.com/apache/incubator-zeppelin/blob/master/docs/interpreter/alluxio.md) 
```

Comment:
alluxio.md -> tachyon.md so that the link works for now. Once we update zeppelin we can change this to alluxio.md